packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -313,13 +314,20 @@ class TextStyle extends Diagnosticable {
   /// parent text span, or, for the root text spans, with the line box.
   final TextBaseline textBaseline;
 
-  /// The height of this text span, as a multiple of the font size.
+  /// The height of this text span, as a multiple of the sum of font size and leading.
   ///
   /// If applied to the root [TextSpan], this value sets the line height, which
   /// is the minimum distance between subsequent text baselines, as multiple of
   /// the font size.
```

Comment:
either `font size and leading` or `font size (the root [TextSpan] ignores the [leading])`

Reply:
looks like the latter